### PR TITLE
feat(osp): created new ui screens

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
## Description

Created new UI as per figma designs

## Why

The current existing prototype implementation of the Onboarding Service Provider flow is supposed to get updated based on the final design.

## Issue

https://github.com/eclipse-tractusx/portal-frontend/issues/483

## Checklist

Please delete options that are not relevant.

- [X] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [X] I have performed a self-review of my own code
- [X] I have successfully tested my changes locally

### CODE REVIEW

File: src/assets/locales/de/registration.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "osp": {
+    "heading": "Registration Summary",
+    "companyHeading": "Company Details",
+    "companyMessage": "Below you can find your company registration details shared by your onboarding provider. Please review the details and proceed to the Company Role selection below before submitting your registration.",
```
Comment (manojava-gk) [inline src/assets/locales/de/registration.json:5]: why not using translator approach here?

File: src/components/pages/OSPConsent/style.scss [hunk 2]
```diff
@@ -0,0 +1,186 @@
+/********************************************************************************
+ * Copyright (c) 2021, 2023 BMW Group AG
+ * Copyright (c) 2021, 2023 Contributors to the Eclipse Foundation
```
Comment (manojava-gk) [inline src/components/pages/OSPConsent/style.scss:3]: use latest template header

File: src/components/shared/frame/Header/RegistrationReviewOverlay/RegistrationReviewContent/RegistrationStatusList.tsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
+/********************************************************************************
+ * Copyright (c) 2021, 2023 BMW Group AG
+ * Copyright (c) 2021, 2023 Contributors to the Eclipse Foundation
```
Comment (manojava-gk) [inline src/components/shared/frame/Header/RegistrationReviewOverlay/RegistrationReviewContent/RegistrationStatusList.tsx:3]: use latest template header

**Review by manojava-gk — COMMENTED** [review]
Along with these small findings, make sure that there is zero linter issue in newly created files.

**Comment by nidhigarg-bmw** [comment]
> Along with these small findings, make sure that there is zero linter issue in newly created files.

@manojava-gk Already checked zero linters and made these changes as well

**Review by manojava-gk — APPROVED** [review]
lgtm

File: src/assets/locales/de/registration.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
+{
+  "osp": {
+    "heading": "Registrierungszusammenfassung",
+    "companyHeading": "Firmendetails",
+    "companyMessage": "Unten finden Sie die Registrierungsdaten Ihres Unternehmens, die Ihnen von Ihrem Onboarding-Anbieter mitgeteilt wurden. Bitte überprüfen Sie die Details und fahren Sie mit der Auswahl der Unternehmensrolle unten fort, bevor Sie Ihre Registrierung absenden.",
+    "companyName": "Name der Firma",
+    "bmw": "BMW AG",
+    "street": "Hausnummer",
+    "zip": "Reißverschluss",
```
Comment (Leon-Gr) [inline src/assets/locales/de/registration.json:9]: Reißverschluss -> Postleitzahl

File: src/assets/locales/de/registration.json [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    "city": "Stadt",
+    "region": "Region",
+    "country": "Land",
+    "companyRole": {
+      "title": "Rollenauswahl",
+      "subTitle": "Hinweis: Bitte wählen Sie mindestens eine Rolle aus und akzeptieren Sie die zugehörigen Geschäftsbedingungen.",
+      "TermsAndCondSpan1": "Ja, ich stimme dem zu",
```
Comment (Leon-Gr) [inline src/assets/locales/de/registration.json:16]: assuming this should be the same as in English, i needs to be "Ja, ich stimme den"

File: src/assets/locales/de/registration.json [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+      "TermsAndCondSpan2": "Geschäftsbedingungen",
+      "TermsAndCondSpan3": "der ausgewählten Rolle.",
```
Comment (Leon-Gr) [inline src/assets/locales/de/registration.json:18]: "der ausgewählten Rolle zu."

File: src/assets/locales/de/registration.json [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+      "TermsAndCond2Span1": "Ja, ich stimme dem zu",
```
Comment (Leon-Gr) [inline src/assets/locales/de/registration.json:19]: same again "Ja, ich stimme den"

File: src/assets/locales/de/registration.json [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+      "TermsAndCond2Span2": "Sicherheitsvorschriften",
+      "TermsAndCond2Span3": "der ausgewählten Rolle.",
```
Comment (Leon-Gr) [inline src/assets/locales/de/registration.json:21]: "der ausgewählten Rolle zu."

File: src/assets/locales/de/registration.json [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+      "ACTIVE_PARTICIPANT": "Aktiver Teilnehmer",
+      "APP_PROVIDER": "App-Anbieter",
+      "SERVICE_PROVIDER": "Dienstleister",
```
Comment (Leon-Gr) [inline src/assets/locales/de/registration.json:24]: Dienstleister -> Service-Anbieter

File: src/assets/locales/de/registration.json [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+      "NoRolesError2": " oder wenden Sie sich an den Systemadministrator.",
+      "NoRolesError": "Es ist ein Fehler aufgetreten. Keine Rollen gefunden. Bitte {{ retry }} oder wenden Sie sich an den Systemadministrator.",
+      "Please": "Bitte",
+      "Retry": "wiederholen"
+    },
+    "success": {
+      "heading": "Herzlichen Glückwunsch, Ihre Registrierung war erfolgreich!",
+      "description": "Wir werden Ihre Registrierung nun validieren.<br/> Der Vorgang kann ein paar Stunden dauern. In der Zwischenzeit sollten Sie sich vielleicht bereits einige Einführungsseiten <br/> ansehen, beachten Sie jedoch, dass der gesamte Datenbereich nur <br/> funktioniert verfügbar, sobald die Registrierung erfolgreich abgeschlossen wurde. <br/> Wenn Sie Fragen zum Registrierungsstatus haben, finden Sie den aktuellen Status<br/>unten.",
+      "stepTitle": "Status:",
+      "homepage": "Komm nach Hause"
```
Comment (Leon-Gr) [inline src/assets/locales/de/registration.json:36]: Komm nach Hause -> Startseite

**Review by oyo — APPROVED** [review]

**Pull request merged**